docs/INSTALL.md

```diff
@@ -53,6 +53,9 @@ See [AWS Credentials](CONFIGURATION.md#aws-credentials) for instructions and dif
 
 You may deploy the Mountpoint for Amazon S3 CSI Driver via [Amazon EKS managed add-on](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html#workloads-add-ons-available-eks), [Helm](https://helm.sh/), or [Kustomize](https://github.com/kubernetes-sigs/kustomize).
 
+> [!IMPORTANT]
+> When upgrading the CSI Driver, existing [Mountpoint Pods](ARCHITECTURE.md#the-mounter-component--mountpoint-pod-aws-s3-csi-mounter) remain unchanged to avoid disrupting your active workloads. New Mountpoint Pods with the updated version are created only when workload pods are started or restarted.
```

Comment:
Is it worth sharing a command to verify which pods are running with which Mountpoint version on the cluster? Is this possible today with the current annotations?